src/Scripting/Compiler/Instruction.hpp

```
@@ -0,0 +1,378 @@
+			setOperandValue();
+		}
+
+
+		void setOperandValue() {
+			if (
+				// Operand is std::nullopt
+				!operand.has_value() ||
+
+				// Operand has already been set to a definitive value
+				(operand.has_value() && typeid(operand.value()) != typeid(std::remove_reference_t<decltype(SymbolicInstruction::operand.value())>))
+			)
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)